src/main/java/de/tum/in/www1/artemis/repository/FeedbackConflictRepository.java

```diff
@@ -0,0 +1,26 @@
+package de.tum.in.www1.artemis.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.FeedbackConflict;
+
+/**
+ * Spring Data JPA repository for the FeedbackConflict entity.
+ */
+@Repository
+public interface FeedbackConflictRepository extends JpaRepository<FeedbackConflict, Long> {
+
+    @Query("select distinct conflict from FeedbackConflict conflict where conflict.conflict = true and (conflict.firstFeedback.id in (:feedbackIds) or conflict.secondFeedback.id in (:feedbackIds))")
+    List<FeedbackConflict> findAllByFeedbackList(@Param("feedbackIds") List<Long> feedbackIds);
+
+    @Query("select distinct conflict from FeedbackConflict conflict where conflict.conflict = true and "
```

Comment:
Maybe it would be a good idea to start using the text blocks @MaisiKoleni was referencing today in the meeting as this is a rather long query

Reply:
in a follow up, yes